playground/eShopLite/CatalogService/Program.cs

```diff
@@ -1,10 +1,19 @@
 using CatalogDb;
 using CatalogService;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<CatalogDbContext>("catalogdb");
+
+// Configure Npgsql.EntityFrameworkCore.PostgreSQL services
+if (builder.Configuration.GetConnectionString("catalogdb") is string connectionString)
```

Comment:
So is this just an alternative approach for configuring the connection string that the components use?